src/assisted_test_infra/download_logs/__main__.py

```diff
@@ -39,7 +62,7 @@ def main():
         for cluster in clusters:
             if args.download_all or should_download_logs(cluster):
                 download_cluster_logs(
-                    client, cluster, args.dest, args.must_gather, args.update_by_events, pull_secret=args.pull_secret
+                    client, cluster, args.dest, args.must_gather, args.update_by_events, args.pull_secret
```

Comment:
That will pass `pull_secret` as `retry_interval` argument

Reply:
mistake sorry